quickcheck-dynamic/test/Spec/DynamicLogic/RegistryModel.hs

```diff
@@ -267,6 +267,7 @@ tests =
   testGroup
     "registry model example"
     [ testProperty "prop_Registry" $ prop_Registry
+    , testProperty "prop_Registry . getMoreActions" $ prop_Registry . getMoreActions
```

Comment:
It's good to have a test using that new wrapper, but it would be even better to have a test demonstrating precisely what this new wrapper provides, eg. some test showing the generated test sequences are longer, while the size of generated values are unchanged for example?

Reply:
I've introduced a test for this that uses quickcheck coverage testing. Good idea.